src/service_daemon.rs

```diff
@@ -910,6 +1077,11 @@ struct Zeroconf {
     /// Socket for signaling.
     signal_sock: UdpSocket,
 
+    /// Timestamps marking where we need another iteration of the run loop,
+    /// to react to events like retransmissions, cache refreshes, interface IP address changes, etc.
+    ///
+    /// When the run loop goes through a single iteration, it will
+    /// set its timeout to the earliest timer in this list.
     timers: Vec<u64>,
```

Comment:
This part took me some time to understand, so I've added a comment. I'm still not 100% sure that I understood it properly, so I want to point it out for discussion. Please suggest a better description or corrections if you have any